packages/airnode-protocol/test/rrp/requesters/RrpBeaconServerWithManager.sol.js

```diff
@@ -746,7 +746,12 @@ describe('readBeacon', function () {
           { gasLimit: 500000 }
         );
       // Read the beacon again
-      const currentBeacon = await rrpBeaconServer.connect(roles.beaconReader).readBeacon(templateId);
+      let currentBeacon = await rrpBeaconServer.connect(roles.beaconReader).readBeacon(templateId);
+      expect(currentBeacon.value).to.be.equal(decodedData);
+      expect(currentBeacon.timestamp).to.be.equal(nextBlockTimestamp);
+      // Read beacon with address(0)
+      const voidSigner = new hre.ethers.VoidSigner(hre.ethers.constants.AddressZero, hre.ethers.provider);
+      currentBeacon = await rrpBeaconServer.connect(voidSigner).readBeacon(templateId);
```

Comment:
VoidSigner is not a whitelisted user so this is misplaced. I will push a fix.

Reply:
Done in https://github.com/api3dao/airnode/pull/692/commits/7eda0fdade32fef33d8e295a5172d28dcb0ce35e